 <launch>
    <arg name="model" default="burger" doc="model type [burger, waffle]"/>
    <arg name="x_pos" default="0.0"/>
    <arg name="y_pos" default="0.0"/>
    <arg name="z_pos" default="0.0"/>
    <arg name="gui" default="false"/>
    <rosparam param="sim">false</rosparam>
    <rosparam param="map">false</rosparam>
    <rosparam param="use_tf">true</rosparam>
    <rosparam param="rviz">true</rosparam>

    <node pkg="asl_turtlebot" type="detector.py" name="turtlebot_detector" />
    <node pkg="asl_turtlebot" type="pose_controller.py" name="turtlebot_pose_controller" />
    <node pkg="asl_turtlebot" type="gazebo_plot.py" name="gazebo_plot" />

</launch>
